src/website/webview/templates/components/suggestion.html

```diff
@@ -38,6 +38,25 @@
       {% selectable_nixpkgs_package_list cached_suggestion.packages %}
     {% endif %}
 
+    {% if cached_suggestion.maintainers %}
+      <details class="maintainers">
+        <summary>
+          {{ cached_suggestion.maintainers | length }} package maintainer(s) would be pinged
```

Comment:
```suggestion
          Notify package maintainers: {{ cached_suggestion.maintainers | length }}
```

Before going full gettext we should just try to avoid any sort of grammar... I know this ugly but among all cosmetics this scales better until we have the time for polishing.